playbooks/dbgsym-kernel.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: add debug symbol archive key
+  apt_key: keyserver=keyserver.ubuntu.com id=2512191FEF8729D6E5AF414DECDCAD72428D7C01
```

Comment:
So we can extend the apt_repos role to take in a `keyserver` and `id` as well, but I'm concerned as to whether that will work through the proxy system we have setup.